src/main/webapp/app/oncokb-frontend-commons/src/components/RefComponent.tsx

```diff
@@ -18,8 +18,17 @@ export default class RefComponent extends React.Component<{
       return <span>{this.props.content}</span>;
     }
 
-    const abstractParts = parts[1].split(/(?=http)/i);
-    const isAbstract = !(abstractParts.length < 2);
+    // Slicing from index 1 to end to rejoin the abstract title and links
+    // when there's the case that they also contain the string 'Abstract'
+    // Example :
+    //     (Abstract: Fakih et al. Abstract# 3003, ASCO 2019.)
+    const abstractParts = parts
+      .slice(1)
+      .join('Abstract')
+      .split(/(?=http)/i);
+    const isAbstract =
+      !(abstractParts.length < 2) &&
+      this.props.content.toLowerCase().includes('abstract');
```

Comment:
Do we still need this check?

Reply:
Yeah makes sense to remove `this.props.content.toLowerCase().includes('abstract')` check.